File: src/main/java/top/zbeboy/isy/web/bean/tree/TreeBean.java
package top.zbeboy.isy.web.bean.tree;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.List;

/**
 * Created by lenovo on 2016-10-18.
 */
@AllArgsConstructor(access = AccessLevel.PUBLIC)
@Data
public class TreeBean {
    private String text;
    private List<TreeBean> nodes;
    private int dataId;
}

File: src/main/java/top/zbeboy/isy/web/bean/tree/TreeBean.kt
package top.zbeboy.isy.web.bean.tree

/**
 * Created by zbeboy 2017-11-17 .
 **/
open class TreeBean {
    var text: String? = null
    var nodes: List<TreeBean>? = null
    var dataId: String? = null

    constructor()

    constructor(text: String?, nodes: List<TreeBean>?, dataId: String?) {
        this.text = text
        this.nodes = nodes
        this.dataId = dataId
    }
}